src/pepr/operator/README.md

```diff
@@ -155,19 +155,13 @@ spec:
           bearer_only: clientField(bearerOnly)
 ```
 
-### Controlling how UDS Operator interacts with Keycloak
+### How UDS Operator interacts with Keycloak
 
-The UDS Operator can interact with Keycloak in two primary ways: using the Dynamic Client Registration or Client Credentials Grant. The method used is determined automatically or by specifying the environment variable `PEPR_KEYCLOAK_CLIENT_STRATEGY`.
+The UDS Operator authenticates with Keycloak using the OAuth 2.0 Client Credentials Grant. It uses the OAuth 2.0 Client Credentials Grant to authenticate against the `uds-operator` client defined in Keycloak. This special client has a limited control over managing Keycloak Clients for the UDS Operator.
```

Comment:
```suggestion
The UDS Operator authenticates with Keycloak using the OAuth 2.0 Client Credentials Grant. It authenticates against the `uds-operator` client defined in Keycloak, which has limited control to manage Keycloak Clients for UDS.
```

This wording felt weird/redundant. Not sure if my suggestion is better, just tried to simplify a bit?